roles/vdbench/templates/client.yaml

```diff
@@ -0,0 +1,68 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: 'vdbench-client-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+spec:
+  ttlSecondsAfterFinished: 600
+  template:
+    metadata:
+      labels:
+        app: vdbenchd-client-{{ trunc_uuid }}
+    spec:
+      containers:
+      - name: vdbench-client
+        #image: "docker-registry.upshift.redhat.com/vdbench/vdbench:master"
+        image: "quay.io/avili/vdbench:1.0.4"
```

Comment:
if the image is available from quay.io then anyone in the world can pull it, right?    This contradicts the terms of the Oracle license.   I suggest you create a RedHat-internal docker image repo (not hard to do) and use that here.   A customer or partner that wants to use it will have to rebuild the image themselves and change the image: field to point to theirs.   Sorry, I think their license is stupid but that's what it says.